generate_dip.py

```diff
@@ -103,15 +106,25 @@ def _uuid(identifier: str) -> str:
 
         uuid_pkg = _uuid('pkg')
         uuid_pads = [_uuid('pad-{}'.format(p)) for p in range(1, pin_count + 1)]
-
+        
+        # IPC-7251 name  Example: DIP762W52P254L1905H508Q14B
+        # DIP + width + W lead_width + P spacing + L pin_package_offset*2 +(pin_count/2)* spacing + H component_height + Q pin_count B
```

Comment:
Would be great if you could use IPC terminology (like "lead span") so that the terms we use are consistent across generators.

```suggestion
        # DIP + lead span + W lead width + P pitch + L body length + H component height + Q pin count B
```